driver-core/src/main/com/mongodb/internal/operation/ServerVersionHelper.java

```diff
@@ -31,20 +31,13 @@ public final class ServerVersionHelper {
     public static final int FOUR_DOT_FOUR_WIRE_VERSION = 9;
     public static final int FIVE_DOT_ZERO_WIRE_VERSION = 12;
     public static final int SIX_DOT_ZERO_WIRE_VERSION = 17;
-    private static final int SEVEN_DOT_ZERO_WIRE_VERSION = 21;
-
-    public static boolean serverIsAtLeastVersionFourDotTwo(final ConnectionDescription description) {
-        return description.getMaxWireVersion() >= FOUR_DOT_TWO_WIRE_VERSION;
-    }
+    public static final int SEVEN_DOT_ZERO_WIRE_VERSION = 21;
+    public static final int LATEST_WIRE_VERSION = SEVEN_DOT_ZERO_WIRE_VERSION;
```

Comment:
Added this so we don't have to update the tests again every time we bump the minWireVersion